ddtrace/internal/module.py

```diff
@@ -27,8 +34,6 @@
 log = get_logger(__name__)
 
 ModuleHookType = Callable[[ModuleType], None]
```

Comment:
does this need to move too?

Reply:
We import it at ``ddtrace.debugging._debugger``, so it cannot be under the if (unless we change it there too)

Reply:
oh, I see we are keeping `ModuleType`, and `Callable` not in the if statement 👍🏻